### ISSUE TITLE
History is not compatible with prompt_toolkit 2.0

### ISSUE THREAD

**Issue by theazureshadow** [click-contrib/click-repl#43]
Just a heads up: `prompt_toolkit` just released a [2.0 version](https://github.com/jonathanslenders/python-prompt-toolkit/releases/tag/2.0.1). Not sure if it'll break anything else, but it broke history for us. It's possible this actually represents a bug in prompt_toolkit code rather than click-repl, but I haven't dug into it. [The docs](http://python-prompt-toolkit.readthedocs.io/en/stable/pages/reference.html#prompt_toolkit.shortcuts.prompt) still imply that `history` should be a keyword argument.

We pinned our version to 1.x for now.

Here's the stack:

```
  File "/usr/local/lib/python2.7/site-packages/click_repl/__init__.py", line 192, in repl
    command = get_command()
  File "/usr/local/lib/python2.7/site-packages/click_repl/__init__.py", line 186, in get_command
    return prompt(**prompt_kwargs)
  File "/usr/local/lib/python2.7/site-packages/prompt_toolkit/shortcuts/prompt.py", line 793, in prompt
    return session.prompt(*a, **kw)
TypeError: prompt() got an unexpected keyword argument 'history'
```

**Comment by theazureshadow** [comment]
Opened an issue in prompt toolkit: https://github.com/jonathanslenders/python-prompt-toolkit/issues/632

**Comment by theazureshadow** [comment]
The prompt toolkit issue is fixed in master. Now for the `prompt` shortcut, they'll pop the `history` argument out of `kwargs` and pass it to `PromptSession`. It hasn't made it into a release yet.

**Comment by untitaker** [comment]
Probably fixed by https://github.com/click-contrib/click-repl/commit/62559405f01fb444819beccbb8cd5816b402e132, released in 0.1.5

**Comment by theazureshadow** [comment]
We unpinned the prompt toolkit version, and it still worked. I'm closing this issue.
